Create ocean GitHub action (#1)
port-labs/ocean-github-action: sail/README.md

<!-- action-docs-header action="action.yml" -->

<!-- action-docs-header action="action.yml" -->

<!-- action-docs-description action="action.yml" -->
## Description

> Runs the Ocean Sail command for a specific type of integration

Reviewer: This section either needs a direct link to the docs about the Ocean sail command, or it needs to include a short explanation on what the Ocean sail command does

Author: Added explanation because we dont have this section :<

![image](https://github.com/port-labs/ocean-github-action/assets/63305203/02e89e3d-dbce-4785-8f74-04e2625e846c)

<!-- action-docs-description action="action.yml" -->

Read more about the Ocean framework [here](https://ocean.getport.io/)

<!-- action-docs-inputs action="action.yml" -->
## Inputs

| name | description | required | default |
| --- | --- | --- | --- |
| `type` | <p>The type of the integration to run</p> | `true` | `""` |
| `identifier` | <p>The identifier of the integration to run</p> | `false` | `""` |

> | `port_client_id` | <p>The port client id</p> | `true` | `""` |

Reviewer: ```suggestion
| `port_client_id` | <p>The Port client id</p> | `true` | `""` |
```

> | `port_client_secret` | <p>The port client secret</p> | `true` | `""` |

Reviewer: ```suggestion
| `port_client_secret` | <p>The Port client secret</p> | `true` | `""` |
```

| `initialize_port_resources` | <p>Should ocean try to create the default blueprints, pages &amp; integration config for the integration</p> | `false` | `true` |
| `config` | <p>The configuration for the integration</p> | `false` | `""` |
| `platform` | <p>The platform to run the integration on</p> | `false` | `linux/amd64` |
| `image` | <p>The image to run the integration from</p> | `false` | `""` |
| `version` | <p>The version of the integration to run</p> | `false` | `latest` |
<!-- action-docs-inputs action="action.yml" -->

<!-- action-docs-outputs action="action.yml" -->
## Outputs

| name | description |
| --- | --- |
| `exit_code` | <p>The exit code of the Ocean Sail command</p> |
<!-- action-docs-outputs action="action.yml" -->

## Available integration types

> All available integration types are listed in the [Ocean documentation](https://ocean.getport.io/integrations-library/) and also can be found when trying to add a new Datasource in the Port UI.

Reviewer: ```suggestion
All available integration types are listed in the [Ocean integrations library](https://ocean.getport.io/integrations-library/) and can also be found in the data sources menu in the Port UI.
```
